tests/unit/models/test_util.py

```diff
@@ -40,6 +45,27 @@ def test_exponential_counter__reset(self):
             counter.reset()
 
 
+class TestBoundedSet(UnitTest):
+    def test_contains(self):
+        bset = BoundedSet(max_items=10)
+        bset.add(1)
+        assert 1 in bset
+
+    def test_bound(self):
+        bset = BoundedSet(max_items=10)
+        [bset.add(i) for i in range(11)]
+        assert len(bset._set) == 10
+        assert 0 not in bset
+
+    def test_lru(self):
```

Comment:
Perhaps we should have two versions of this test: the current version shows that `add` marks the element as recently used, but it's also important that `__contains__` does.